examples/Preprocessing_example.md

```diff
@@ -0,0 +1,77 @@
+# Preprocessign an Image
+
+The main objective of the Preprocessing phase is to make as easy as possible for the OCR system to distinguish a character/word from the background. There are several techniques used for Preprocessing but this **Preprocessor** class can do it without you going into details and jargon of traditional methods.
+
+`!pip install -U ocred --no-cache --force-reinstall`
+
+```
+# manually preprocessing an image
+import cv2
+from scipy import ndimage
+from ocred import Preprocessor
+from google.colab.patches import cv2_imshow
+
+
+preprocessed = Preprocessor("/content/images/CosmosOne.jpg")
```

Comment:
Most of the users will be using `Jupyter` locally. You can make this change at the end, when you are done with running this code on colab!
```suggestion
preprocessed = Preprocessor("../images/CosmosOne.jpg")
```